### ISSUE TITLE
onSuccess doesn't get called.

### ISSUE THREAD

**Issue by Avrohom** [AnthonyNahas/ngx-auth-firebaseui#155]
### Bug Report or Feature Request (mark with an `x`)

- [x ] bug report -> please search issues before submitting
- [ ] feature request

### OS and Version?
 Windows 10. 

### Versions
     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/

Angular CLI: 7.1.3
Node: 8.11.1
OS: win32 ia32
Angular: 7.1.3
... animations, cli, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.11.3
@angular-devkit/build-angular     0.11.3
@angular-devkit/build-optimizer   0.11.3
@angular-devkit/build-webpack     0.11.3
@angular-devkit/core              7.1.3
@angular-devkit/schematics        7.1.3
@angular/cdk                      7.1.1
@angular/fire                     5.1.1
@angular/flex-layout              7.0.0-beta.19
@angular/material                 7.1.1
@ngtools/webpack                  7.1.3
@schematics/angular               7.1.3
@schematics/update                0.11.3
rxjs                              6.3.3
typescript                        3.1.6
webpack                           4.23.1

### Repro steps

<ngx-auth-firebaseui *ngIf="thisUser==null"
                     [guestEnabled]="false"
                     (onSuccess)="printUser($event)"
                     (onError)="printError($event)">
</ngx-auth-firebaseui>
<ngx-auth-firebaseui-user *ngIf="thisUser"></ngx-auth-firebaseui-user>

### Desired functionality

I would like to see that the onSuccess gets called.

### Mention any other details that might be useful

onSuccess doesn't get called. On the above sample, onSuccess does nothing and onError will send the error object 2 more times to the console. So without the above onError call, error event will show TWICE and WITH the onError call it will show FOUR TIMES on each unsuccessful login attempt.

Please note: this is not the same issue as #138 'Progress bar doesn't stop and onSuccess doesn't work', because in my case the progress-bar does stop. Also, the firebase User object gets updated after a successful login.

**Comment by AnthonyNahas** [comment]
@Avrohom which version are you using ?

**Comment by Avrohom** [comment]
The latest.

**Comment by AnthonyNahas** [comment]
@Avrohom this has been fixed in [v2.3.0](https://www.npmjs.com/package/ngx-auth-firebaseui) which is live now

**Comment by kashalkar** [comment]
Hi Anthony,

Sorry, I had to reopen this as I face the same issue with v2.7.2. OnSuccess not called. 
Tried the workaround above...but did not help either.

app.module.ts
```
NgxAuthFirebaseUIModule.forRoot(
      firebaseConfig,
      () => 'abcdef',
      {
        enableFirestoreSync: false,
        toastMessageOnAuthSuccess: true,
        toastMessageOnAuthError: true
      }
    ),
```

package.json

   "@angular/animations": "^7.2.15",
    "@angular/cdk": "^7.3.7",
    "@angular/common": "~7.2.0",
    "@angular/compiler": "~7.2.0",
    "@angular/core": "~7.2.0",
    "@angular/fire": "^5.1.3",
    "@angular/flex-layout": "^7.0.0-beta.23",
    "@angular/forms": "~7.2.0",
    "@angular/material": "^7.3.7",
    "@angular/platform-browser": "~7.2.0",
    "@angular/platform-browser-dynamic": "~7.2.0",
    "@angular/pwa": "^0.12.4",
    "@angular/router": "~7.2.0",
    "@angular/service-worker": "~7.2.0",
    "@material/layout-grid": "^0.41.0",
    "@ngx-translate/core": "^11.0.1",
    "@ngx-translate/http-loader": "^4.0.0",
    "core-js": "^2.5.4",
    "firebase": "^6.0.4",
    "hammerjs": "^2.0.8",
    "ngx-auth-firebaseui": "^2.7.2",
    "rxjs": "~6.3.3",
    "tslib": "^1.9.0",
    "zone.js": "~0.8.26"

Do you have any ideas?

Thanks
